### PR TITLE
D8CORE-3493 D8CORE-3513 Wysiwyg video style fixes

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Make the wrapper of an iframe that has a caption 100%.
- don't allow floating of a video.

# Need Review By (Date)
- 2/25

# Urgency
- Medium

# Steps to Test
1. checkout this branch & clear browser cache
2. create a basic page and embed a youtube video in a wysiwyg
3. set a caption and save
4. make sure the video isn't tiny
5. remove the caption and set the video to align right or left & save.
6. make sure the video still displays.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by jenbreese** [comment]
It isn't working as I think it should.
With a caption:
Screen Shot 2021-02-17 at 3 53 57 PM

Without a caption:
Screen Shot 2021-02-17 at 3 54 59 PM

**Comment by pookmish** [comment]
@jenbreese I might have forgot to mention clear drupal caches just in case

**Comment by jenbreese** [comment]
@pookmish I did but will try again. :)

**Comment by jenbreese** [comment]
Clearing the cache fixed it. The video is 100% and doesn't move to the left or right but that should be in another ticket.

File: lib/js/jumpstart_ui.behavior.js [hunk 1]
```diff
@@ -10,12 +10,16 @@
 export default {
 
   // Attach Drupal Behavior.
-  attach (context, settings) {
-    // console.log("Attached.");
+  attach(context, settings) {
+    (function ($) {
+      $('figure .media-entity-wrapper.video', context).each((i, video) => {
+        $(video).closest('figure').css('width', '100%');
+      })
```
Comment (jenbreese) [inline lib/js/jumpstart_ui.behavior.js]: Do you need a ; here?

Comment (pookmish) [reply lib/js/jumpstart_ui.behavior.js]: Don't really _need_ one. Yay for javascript being the most unstructured code. :rofl:

Comment (jenbreese) [reply lib/js/jumpstart_ui.behavior.js]: Well. True. But we usually do have one. :D

Comment (pookmish) [reply lib/js/jumpstart_ui.behavior.js]: You got it :+1:

Comment (jenbreese) [reply lib/js/jumpstart_ui.behavior.js]: Thank you!

**Review by jenbreese — APPROVED** [review]
Looks good to me

**Pull request merged**